tests/rule_based_profiler/conftest.py

```diff
@@ -1518,6 +1518,137 @@ def bobster_columnar_table_multi_batch_normal_mean_5000_stdev_1000_data_context(
     return context
 
 
+@pytest.fixture
+def quentin_columnar_table_multi_batch():
+    """
+    About the "Quentin" User Workflow Fixture
+
+    Quentin has multiple tables of columnar data called user_events (DataAsset) that he wants to check periodically as
+    new data is added.
+
+      - He knows what some of the columns are of the acconting/financial/account type, but he is currently interested in
```

Comment:
Typo in "accounting"

Reply:
@NathanFarmer Nice catch!!!  Fixed!  Thank you!